front/temporal/agent_loop/lib/run_model.ts

```diff
@@ -463,32 +472,25 @@ export async function runModelActivity(
       prompt,
     });
   } else {
-    if (userMessage.rank === 0) {
-      // Log conversations that are using the new LLM router (log only once when the conversation starts)
-      localLogger.info(
-        {
-          conversationId: conversation.sId,
-        },
-        "Running model with the new LLM router"
-      );
-    }
-    getOutputFromActionResponse = await getOutputFromLLMStream(auth, {
-      modelConversationRes,
-      conversation,
-      userMessage,
-      runConfig,
-      specifications,
-      flushParserTokens,
-      contentParser,
-      agentMessageRow,
-      step,
-      agentConfiguration,
-      agentMessage,
-      model,
-      publishAgentError,
-      prompt,
-      llm,
-    });
+    // This returns the old implementation's response for now
+    getOutputFromActionResponse =
```

Comment:
and the tldr; of the PR, we add a helper function to run both models and do the comparaison